networks/movement/movement-client/src/bin/e2e/load_alice_bob.rs

```diff
@@ -86,107 +88,71 @@ static FAUCET_URL: Lazy<Url> = Lazy::new(|| {
 
 #[async_trait::async_trait]
 impl Scenario for BasicScenario {
-	async fn run(self: Box<Self>) -> Result<()> {
+	async fn prepare(&mut self) -> Result<(), anyhow::Error> {
 		let rest_client = Client::new(NODE_URL.clone());
 		let faucet_client = FaucetClient::new(FAUCET_URL.clone(), NODE_URL.clone());
-		let coin_client = CoinClient::new(&rest_client);
+		let coin_client = CoinClient::new(&rest_client); // Print initial balances.
```

Comment:
Does this comment relate to the line? It seems spuriously added.

Reply:
Yes it's an old comment not related to the line. Should be removed.

Reply:
Update done with suggestion.